app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.java

```diff
@@ -114,6 +116,8 @@ public void onPageScrollStateChanged(int state) {
 
       }
     });
+
+    Log.i("kiwik", "ZimManageActivity successfully bootstrapped");
```

Comment:
kiwik? poor thing's got a speech impediment?  There's a typo in case you didn't spot it BTW :) 

And also, how about having a single const value in this file for the tag and making the tag reflect the activity (ZimManageActivity). Perhaps it'd be worth considering a naming convention for tags e.g. one suggestion is to start each tag with `Kiwix` followed by a separator e.g. `-` and then the purpose of the Java class? 

If we agree a naming convention would be useful, we can and should revise all the log statements to use the naming convention.